x-pack/plugins/enterprise_search/public/applications/app_search/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { formatApiName } from './format_api_name';
```

Comment:
We should to consider using folder architecture (similar to the `shared/` folder) instead of dumping single files in the `utils/` root, i.e.:

```
- app_search
  |- components/
  |_ utils/
    |- format_api_name/
    | |- index.test.ts
    | |_ index.ts
    |- some_helper/
    |_ some_other_helper/
```

otherwise we end up with a util folder that's twice as long as our current ent-search one, primarily due to extra `.test.ts` files:

```
- app_search
  |- components/
  |_ utils/
    |- format_api_name.test.ts
    |- format_api_name.ts
    |- index.ts
    |- some_helper.test.ts
    |- some_helper.ts
    |- some_other_helper.test.ts
    |_ some_other_helper.ts
```

Which is slightly to read/navigate about, just IMO.

I know that changes imports to `import { formatApiName } from '../utils/format_api_name';` vs just `from '../utils';` but IMO that's an OK tradeoff (and we can maintain a `utils/index.ts` with exports if we really really want 🤷)

Reply:
c80e0c89b6c99286db668d69da57bcc62e9f92a0